crates/bevy_pbr/src/render/mesh_view_bindings.wgsl

```diff
@@ -43,15 +43,17 @@ var<uniform> cluster_offsets_and_counts: ClusterOffsetsAndCounts;
 
 @group(0) @binding(9)
 var<uniform> globals: Globals;
+@group(0) @binding(10)
+var<uniform> fog: Fog;
 
 #ifdef MULTISAMPLED
-@group(0) @binding(10)
-var depth_prepass_texture: texture_depth_multisampled_2d;
 @group(0) @binding(11)
+var depth_prepass_texture: texture_depth_multisampled_2d;
+@group(0) @binding(12)
 var normal_prepass_texture: texture_multisampled_2d<f32>;
 #else
-@group(0) @binding(10)
-var depth_prepass_texture: texture_depth_2d;
 @group(0) @binding(11)
+var depth_prepass_texture: texture_depth_2d;
+@group(0) @binding(12)
```

Comment:
This is a really weird diff, what I did was actually just change the `@binding` numbers from `10`, `11` to `11`, `12`.